app/handlers/api/applications.go

```diff
@@ -31,3 +32,25 @@ func Applications(w http.ResponseWriter, r *http.Request) {
 	}
 	json.NewEncoder(w).Encode(apps)
 }
+
+// Application provides an Application
+func Application(w http.ResponseWriter, r *http.Request) {
```

Comment:
I think that's a better way to handle this. I didn't know how to feel about the classic `Index`, `Show`, `Create` wording, but I think that it's best for this case.